courses/machine_learning/deepdive2/structured/labs/1_explore_babyweight.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<p>I'd add the following hint into the TODO:</p><p> </p><p>"TODO: Reusing the get_distinct_values function you just implemented,  create function that plots distinct value statistics from BigQuery"</p><p> </p><p>I'd also put the todo into its own markdown cell instead of a code comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/665/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Done</p>